docs/src/continous-integration.rst

```diff
@@ -0,0 +1,94 @@
+*********************************
+Continuous Integration (CircleCI)
+*********************************
+
+.. admonition:: What is continuous integration?
+
+   Continuous integration (CI) is a software development strategy that increases the speed of development while ensuring
+   the quality of the code that teams deploy. Developers continually commit code in small increments (at least daily, or
+   even several times a day), which is then automatically built and tested before it is merged with the shared repository.
+
+   Source: `CircleCI documentation <https://circleci.com/continuous-integration/>`__
+
+Configuration
+=============
+
+Our CircleCI configuration resides in :github-source:`.circleci/config.yml`.
+It contains all the jobs listed below.
+See `Configuring CircleCI <https://circleci.com/docs/2.0/configuration-reference/>`__ for a full reference.
+
+Workflow ``develop``
+====================
+
+This workflow gets triggered everytime a commit is pushed to the ``develop`` branch or a PR is opened.
+
+.. image:: images/circleci-main-workflow.png
```

Comment:
Since the workflows are not identical on integreat-cms and lunes-cms, I suggest to upload these images for lunes:

develop workflow:
![Screenshot 2022-07-15 at 12-23-51 Workflow - lunes-cms_346_e8af93f1-f7ff-4fbd-b808-b798432c3837`](https://user-images.githubusercontent.com/16021269/179205598-fc40057e-d68b-4e1c-b6d3-5507a1922830.png)

main workflow:
![Screenshot 2022-07-15 at 12-26-25 Workflow - lunes-cms_319_e2e04dde-6f85-47f1-9b41-6103c3f97972`](https://user-images.githubusercontent.com/16021269/179205951-8ec1ae35-6669-427a-bc92-b94c4fbe7302.png)

deploy workflow:
![Screenshot 2022-07-15 at 12-27-01 Workflow - lunes-cms_320_1ea9ac9e-9612-48f0-836d-f73a58a3212f`](https://user-images.githubusercontent.com/16021269/179205971-4afde437-a2f3-43e5-9bd7-0cf6058d0637.png)